Docs/Book/Install.md

```diff
@@ -43,42 +43,43 @@ C:\> activate my-rdkit-env
 
 For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit).
 
-#### macOS 10.12 (Sierra): Python 3 environment
+#### macOS 15 (Sequoia): Python 3 environment
 
-The following commands will create a development environment for macOS Sierra and Python 3. Download
-Miniconda3-latest-MacOSX-x86_64.sh from [Conda](http://conda.pydata.org/miniconda.html) and run these
+The following commands has been tested on a Apple M4 chip with MacOS Sequoia (MacOS 15). It is expected to work on other recent macOS versions running on Apple Silicon. For Intel-based macOS systems, please download the according Miniconda installer.
+Download Miniconda from [Conda](https://www.anaconda.com/docs/getting-started/miniconda/install#macos-terminal-installer) and run these
 following commands:
 
 ```
-bash Miniconda3-latest-MacOSX-x86_64.sh
-conda install numpy matplotlib
-conda install cmake cairo pillow eigen pkg-config
-conda install boost-cpp boost py-boost
+bash Miniconda3-latest-MacOSX-arm64.sh # or bash ~/Miniconda3-latest-MacOSX-x86_64.sh for Intel-based systems
+conda create -n my-rdkit-env
+conda activate my-rdkit-env
+conda install compilers libcxx cmake \
+      libboost libboost-devel \
+      libboost-python libboost-python-devel \
+      numpy matplotlib cairo pillow eigen pandas qt
 ```
 
-Optionally, add the following packages to your environment as useful development tools.
 
-```
-pip install yapf==0.11.1
-pip install coverage==3.7.1
-```
-
-Then follow the usual build instructions. The `PYTHON_INCLUDE_DIR` must be set in the
-cmake command.
+Then follow the usual build instructions. 
 
 ```
-PYROOT=<path to miniconda3>
-cmake -DPYTHON_INCLUDE_DIR=$PYROOT/include/python3.6m  \
-  -DRDK_BUILD_AVALON_SUPPORT=ON \
-  -DRDK_BUILD_CAIRO_SUPPORT=ON \
-  -DRDK_BUILD_INCHI_SUPPORT=ON \
+git clone https://github.com/rdkit/rdkit.git
+cd rdkit
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release \
```

Comment:
You are targeting a different type of install here than what the other instructions have. The others are doing in-tree builds - appropriate for people who will be working with the code - while you are installing in the "system" location (the conda dirs in this case) - appropriate for people who just want to use the code.

There are arguments for either approach, but I would prefer that we remain consistent in this document and that the Mac instructions do an in-tree install.